assets/prefabs/structures/Smith.prefab

```diff
@@ -0,0 +1,70 @@
+{
+    "StructureTemplate": {
+    },
+    "SpawnBlockRegions": {
+        "regionsToFill": [
+            { "blockType": "CoreBlocks:DoorBottom.BACK", "region": { "min": [-2, 1, 1], "size": [2, 1, 1]}},
+            { "blockType": "CoreBlocks:DoorTop.BACK", "region": { "min": [-2, 2, 1], "size": [2, 1, 1]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [-4, 2, 3], "size": [1, 1, 2]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [4, 2, 3], "size": [1, 1, 2]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [2, 2, 6], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.BACK", "region": { "min": [-2, 6, 4], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.BACK", "region": { "min": [-3, 5, 5], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.BACK", "region": { "min": [-4, 4, 6], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.FRONT", "region": { "min": [4, 4, 1], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.FRONT", "region": { "min": [3, 5, 2], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.FRONT", "region": { "min": [2, 6, 3], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.LEFT", "region": { "min": [-4, 4, 1], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.LEFT", "region": { "min": [-3, 5, 2], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.LEFT", "region": { "min": [-2, 6, 3], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.RIGHT", "region": { "min": [2, 6, 4], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.RIGHT", "region": { "min": [3, 5, 5], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.RIGHT", "region": { "min": [4, 4, 6], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownRoof.BACK", "region": { "min": [-1, 6, 4], "size": [3, 1, 1]}},
```

Comment:
Equivalently, all occurrences of `HjerlHede:brownRoof` can be replaced by `hjerlhede:brownroof:engine:slope` as mentioned above.

Reply:
I tried that and command above. It didn't work out it is not placing any blocks.
![ss111](https://user-images.githubusercontent.com/44372113/71331603-02035c80-2544-11ea-8478-2340f9fb4d26.png)

Reply:
Hm, that's weird. 🤔 Let's merge it in the current state, we can still improve this later on. 😉